src/FSharp.MetadataFormat/FSharp.MetadataFormat.fsproj

```diff
@@ -119,7 +119,7 @@
     </ProjectReference>
   </ItemGroup>
   <Choose>
-    <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And ($(TargetFrameworkVersion) == 'v4.0')">
+    <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And $(TargetFrameworkVersion) == 'v4.0'">
```

Comment:
@mathhid about the conditions, I was thinking it is already hard to generate those, but we can easily make a distinct (per .proj) on them and give a shot at a "namer" function.

Regarding minimizing the diff, those conditions keep changing, it might change less frequently in future but it will still keep changing, I see it as factorization exercise which will also help users (reviewing *proj changes is no fun and the amount of changed things could be reduced a lot ideally).

Anyways didn't intend to derail from the actual PR matter so we will catch up on Paket repository in the future :)

Thanks both for feedback!